app/models/manageiq/providers/embedded_terraform/provider.rb

```diff
@@ -0,0 +1,20 @@
+class ManageIQ::Providers::EmbeddedTerraform::Provider < Provider
+  include DefaultTerraformObjects
```

Comment:
I'm not sure I understand why this is a module because I don't see it reused anywhere (or do you plan to reuse in the future?  Prefer inlining the methods from the module directly here unless you plan on reuse.

Reply:
Needs review.